backends/dpdk/dpdkProgram.cpp

```diff
@@ -35,13 +35,58 @@ const IR::DpdkAsmStatement *ConvertToDpdkProgram::createListStatement(
     return new IR::DpdkListStatement(name, *stmts);
 }
 
+/* Insert the metadata structure updated with tmp variables created during parser conversion
+   Add annotations to metadata and header structures and add all the structures to DPDK
+   structtype.
+*/
+IR::IndexedVector<IR::DpdkStructType> ConvertToDpdkProgram::UpdateHeaderMetadata
```

Comment:
Does our linter like this? I expect it wants the open parens on the previous row.